config.js

```diff
@@ -397,6 +397,16 @@ config.SEND_EVENTS_REMOTESYS = true;
 config.PROMETHEUS_ENABLED = true;
```

Comment:
Not sure, 
You can just configure Prometheus to scrape only the endpoints for some services/reports

Reply:
Since this config is build time, I agree with Ohad and I Don't see the need to break into different ones.
If we would need it in the future, we'd need to move to env based anyway

Reply:
I just want to add that even if we override the config using an env variable the pod will restart. 
This is the way k8s works

Reply:
That is fine, we might need to be able to be more selective on what we collect in the future (we might not as well). Let's handle it if and when the need arises

Reply:
Ok.